model/gen-ai/registry.yaml

```diff
@@ -332,3 +332,57 @@ groups:
 
           Additional output format details may be recorded in the future in the
           `gen_ai.output.{type}.*` attributes.
+      - id: gen_ai.evaluation.name
+        stability: development
+        type: string
+        brief: The qualified name of the evaluator used to evaluate the GenAI response.
+        examples: ["Relevance", "IntentResolution"]
```

Comment:
nit: the brief mentions that the name should be qualified, but examples are not following the recommendation

Reply:
Have updated the description to avoid this confusion. Please share your feedback.